src/lib/djerba/extract/report_to_json.py

```diff
@@ -309,14 +309,24 @@ def build_genomic_biomarkers(self, input_dir, sample_ID, tmb_value=None, msi_fil
             self.assemble_TMB(tmb_value),
             self.assemble_MSI(msi_file_path)
         ]
+        # write and annotate the biomarkers file in the report directory
+        # the file is used by the build_therapy_info() method
+        # ideally this should done earlier, not in report_to_json.py
+        # but OK for now, because this operation will be refactored into a plugin
         self.logger.debug("Annotating Genomic Biomarkers")
-        genomic_biomarkers_path = self.print_genome_biomarkers_maf(biomarkers, input_dir, sample_ID)
-        genomic_biomarkers_annotated = os.path.join(input_dir, self.GENOMIC_BIOMARKERS_ANNOTATED)
+        genomic_biomarkers_path = \
+            self.print_genome_biomarkers_maf(biomarkers, input_dir, sample_ID)
+        genomic_biomarkers_annotated = \
+            os.path.join(input_dir, self.GENOMIC_BIOMARKERS_ANNOTATED)
         oncokb_annotator(
             sample_ID,
             self.params.get(xc.ONCOTREE_CODE).upper(),
-            input_dir
+            input_dir,
+            cache_params = self.params.get(xc.ONCOKB_CACHE),
```

Comment:
Important -- we need to supply cache params to the OncoKB annotator, otherwise it won't do any caching